lumen/ai/agents.py

```diff
@@ -290,6 +294,33 @@ def invoke(self, messages: list | str):
         self._render_lumen(pipeline)
 
 
+class TableListAgent(LumenBaseAgent):
```

Comment:
I wonder, do we need an agent every time we need a specific task like this, or is there an easier, intelligent way to utilize the LLM?

Reply:
Yes, indeed.